pennylane/wires.py

```diff
@@ -427,11 +448,26 @@ def unique_wires(list_of_wires):
                     "Expected a Wires object; got {} of type {}.".format(wires, type(wires))
                 )
 
+        label_sets = [wire.toset() for wire in list_of_wires]
+        seen_ever = set()
+        seen_once = set()
+
+        # Find unique set in O(n) time.
+        for labels in label_sets:
+            # (seen_once ^ labels) finds all of the unique labels seen once
```

Comment:
do you want intersection `&` here? :thinking:

Reply:
No, I want the disjoint intersection. Any label in `labels` that is also in `seen_once` is now technically seen twice.